scripts/torch

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+server=${1-localhost}
+
+if [[ ! $(go version) =~ go1\.9 ]]; then
+  echo "This script will only work well with local+remote binaries built with go >= 1.9."
+  exit 1
+fi
+
+FLAMEDIR="$GOPATH/src/github.com/brendangregg/FlameGraph"
+
+function install() {
+  cat <<EOF
+go get -u github.com/brendangregg/FlameGraph/...
```

Comment:
Does `go get` not complain about the fact that this repo contains no go code? In any case, it's weird to clone a non-go repo into `GOPATH` (and then to install it onto `$PATH`). I don't have a better idea about where to put it or how to script this, though.